.dev_scripts/test_benchmark_tmux.py

```diff
@@ -0,0 +1,91 @@
+import argparse
+import os
+
+import yaml
+
+
+def parse_args():
+    parser = argparse.ArgumentParser(
+        description='running benchmark regression with tmux')
+    parser.add_argument(
+        '--config',
+        help='test config file path',
+        default='./.dev_scripts/config_list.yaml')
+
+    # runtime setting parameters
+    parser.add_argument(
+        '--session_name', '-s', help='the tmux session name', default='test')
+    parser.add_argument(
+        '--path',
+        help='the running path, e.g., $mmpose',
+        default='~/open_mmlab/mmpose')
+    parser.add_argument('--env', help='the conda environment', default='pt1.6')
+    parser.add_argument(
+        '--partition', help='the partition name', default='openmmlab')
+
+    parser.add_argument('--gpus', help='the total number of GPUs', default=2)
```

Comment:
It's better to support loading settings like `gpus`, `gpus-per-node', 'partition', 'cpus_per_task' for each model separately from the config file. And the augments here only give the default values.